src/module/iotjs_module_adc.c

```diff
@@ -234,10 +234,10 @@ JHANDLER_FUNCTION(Unexport) {
 iotjs_jval_t InitAdc() {
   iotjs_jval_t jadc = iotjs_jval_create_object();
 
-  iotjs_jval_set_method(&jadc, "export", Export);
```

Comment:
``iotjs_jval_set_method`` comes from the C++ to C notations what I did before.

- iotjs: namespace
- jval: return value
- set_method: method or function name.

Do you have any special reason to name it ``iotjs_register_native_handler``